src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -57,9 +57,13 @@
 
     private String resourceRequestMemory;
 
+    private String resourceRequestEphemeral;
```

Comment:
Replace `ephemeral` with `ephemeralStorage` in the PR please.

Reply:
Fix